### ISSUE TITLE
AIT bin tests (Pcap): ait_pcap

### ISSUE THREAD

**Issue by nttoole** [NASA-AMMOS/AIT-Core#258]
This issue will track the testing and resulting issues related to Python3 and the Pcap bin scripts.

**Comment by nttoole** [comment]
ait_pcap.py

Status: Failed->Fixes Applied->Pass

Pcap file: /Users/nttoole/dev/ait/git_repos/ait_020720/cmdhist.pcap

Run and ERROR:
ait_pcap.py: error: unrecognized arguments: help
[venv] MT-202661{nttoole}% ./ait_pcap.py $PCAP --times 
2020-02-25T17:06:25.862 | WARNING  | AIT_ROOT not set.  Defaulting to "/Users/nttoole/Development/ait/git_repos/ait_020720/AIT-Core/ait/core/bin"
. . .
2020-02-25T17:06:26.065 | PROGRAM  | ./ait_pcap.py /Users/nttoole/dev/ait/git_repos/ait_020720/cmdhist.pcap --times
Traceback (most recent call last):
  File "./ait_pcap.py", line 136, in <module>
    main()
  File "./ait_pcap.py", line 123, in main
    for start, stop in times.values()[0]:
TypeError: 'dict_values' object is not subscriptable


Proposed fix:
```
         if len(times) == 1:
-            for start, stop in times.values()[0]:
+            for start, stop in list(times.values())[0]:
                 print('%s - %s' % (start, stop))
```



RUN with ERROR:
[venv] MT-202661{nttoole}% ./ait_pcap.py $PCAP --query
2020-02-25T17:19:31.339 | WARNING  | AIT_ROOT not set.  Defaulting to "/Users/nttoole/Development/ait/git_repos/ait_020720/AIT-Core/ait/core/bin"
. . .
2020-02-25T17:19:31.562 | PROGRAM  | ./ait_pcap.py /Users/nttoole/dev/ait/git_repos/ait_020720/cmdhist.pcap --query
Traceback (most recent call last):
  File "./ait_pcap.py", line 136, in <module>
    main()
  File "./ait_pcap.py", line 105, in main
    starttime = datetime.datetime.strptime(stime, dmc.ISO_8601_Format)
TypeError: strptime() argument 1 must be str, not datetime.datetime



Proposed fix: Convert default values to string format of the date they contain:
```
         '--stime': {
-            'default' : dmc.GPS_Epoch,
+            'default' : dmc.GPS_Epoch.strftime(dmc.ISO_8601_Format),
             'help'    : ('Starting time for desired telemetry range in '
                          'ISO 8601 Format "YYYY-MM-DDThh:mm:ssZ" (default: '
                          '1980-01-06T00:00:00Z)')
         },
```
 
```
         '--etime': {
-            'default' : datetime.datetime.now(),
+            'default' : datetime.datetime.now().strftime(dmc.ISO_8601_Format),
             'help'    : ('Ending time for desired telemetry range in '
                          'ISO 8601 Format "YYYY-MM-DDThh:mm:ssZ" (default: '
                          'the current time; example: 2018-05-23T18:54:31Z)')
```

**Comment by MJJoyce** [comment]
Fixed in #259
